### ISSUE TITLE
Failed To Launch Error on Modern Android & IOS Devices

### ISSUE THREAD

**Issue by JoshuaeKaiser** [hiukim/mind-ar-js#126]
My object tracking application is failing on both IOS and Android phones (Modern with up to date OS, iphone 11 & Glaxy S20). Im guessing this must be an error my side as I purchases the Udemi course and it seems to work for the instructor on his device.

Works fine in the browser as intended. Im accesing on the phones via local host from my dev machine.

Screenshot of my code attached. 

<img width="1792" alt="Screenshot 2022-02-13 at 14 50 49" src="https://user-images.githubusercontent.com/17850490/153769617-fda49f06-e51c-44cb-a08d-391f034477b3.png">

**Comment by hiukim** [comment]
Can you make sure your are using https?

**Comment by JoshuaeKaiser** [comment]
Yep that was the issue, i was serving over HTTP. HTTPs with Ngrok fixed it.
